src/clients/ftp-server.rb

```diff
@@ -689,18 +597,15 @@ def FTPdCMDStartup(options)
         CommandLine.Print(_("Removing FTP daemon from the boot process..."))
         CommandLine.Print("")
         SetEnableService(false)
-      elsif Ops.get(options, "xinetd") != nil
+      elsif !Ops.get(options, "socket").nil?
         CommandLine.Print("")
         # TRANSLATORS: CommandLine header
         CommandLine.Print(String.UnderlinedHeader(_("Start-Up:"), 0))
         CommandLine.Print("")
         # TRANSLATORS: CommandLine progress information
-        CommandLine.Print(_("Start FTP daemon via xinetd"))
+        CommandLine.Print(_("Start FTP daemon via socket"))
         CommandLine.Print("")
         SetStartedViaXinetd(true)
```

Comment:
Shouldn't this call be removed?

Reply:
why it should be removed? It is CLI that inform you that FTP Daemon is starting by socket listening.

Reply:
I mean the `SetStartedViaXinetd(true)`

Reply:
yep, make sense, now it do socket enablement.